spec/views/admin/content_assets/new.html.erb_spec.rb

```diff
@@ -3,6 +3,10 @@
 RSpec.describe "admin/content_assets/new", type: :view do
   before(:each) do
     assign(:content_asset, FactoryBot.build(:content_asset))
+    @folder_options = [
```

Comment:
Just for reference, you can also do `FactoryBot.create_list(:content_page, 2)`

Reply:
As this code is being loaded via rails_helper (and `spec/support/factory_bot.rb`)
```ruby
RSpec.configure do |config|
  config.include FactoryBot::Syntax::Methods
end
```
You don't need to specify `FactoryBot` to use its methods. So this will work too:
```ruby
@folder_options = create_list(:content_page, 2)
```